include/boost/gil/image_processing/numeric.hpp

```diff
@@ -68,139 +72,218 @@ inline void compute_tensor_entries(
-/// d stands for derivative, and x or y stand for direction.
-/// For example, dx stands for derivative (gradient) in horizontal
-/// direction, and ddxx means second order derivative in horizon direction
-/// https://en.wikipedia.org/wiki/Hessian_matrix
-template <typename GradientView, typename OutputView>
-inline void compute_hessian_entries(
-    GradientView dx,
-    GradientView dy,
-    OutputView ddxx,
-    OutputView dxdy,
-    OutputView ddyy)
-{
-    using x_coord_t = typename OutputView::x_coord_t;
-    using y_coord_t = typename OutputView::y_coord_t;
-    using pixel_t = typename std::remove_reference<decltype(ddxx(0, 0))>::type;
-    using channel_t = typename std::remove_reference<
-                        decltype(
-                            std::declval<pixel_t>().at(
-                                std::integral_constant<int, 0>{}
-                            )
-                        )
-                       >::type;
-
-    constexpr double x_kernel[3][3] =
-    {
-        {1, 0, -1},
-        {2, 0, -2},
-        {1, 0, -1}
-    };
-    constexpr double y_kernel[3][3] =
-    {
-        {1, 2, 1},
-        {0, 0, 0},
-        {-1, -2, -1}
-    };
-    constexpr auto chosen_channel = std::integral_constant<int, 0>{};
-    for (y_coord_t y = 1; y < ddxx.height() - 1; ++y)
-    {
-        for (x_coord_t x = 1; x < ddxx.width() - 1; ++x)
-        {
-            pixel_t ddxx_i;
-            static_transform(ddxx_i, ddxx_i,
-                [](channel_t) { return static_cast<channel_t>(0); });
-            pixel_t dxdy_i;
-            static_transform(dxdy_i, dxdy_i,
-                [](channel_t) { return static_cast<channel_t>(0); });
-            pixel_t ddyy_i;
-            static_transform(ddyy_i, ddyy_i,
-                [](channel_t) { return static_cast<channel_t>(0); });
-            for (y_coord_t y_filter = 0; y_filter < 2; ++y_filter)
-            {
-                for (x_coord_t x_filter = 0; x_filter < 2; ++x_filter)
-                {
-                    auto adjusted_y = y + y_filter - 1;
-                    auto adjusted_x = x + x_filter - 1;
-                    ddxx_i.at(std::integral_constant<int, 0>{}) +=
-                        dx(adjusted_x, adjusted_y).at(chosen_channel)
-                        * x_kernel[y_filter][x_filter];
-                    dxdy_i.at(std::integral_constant<int, 0>{}) +=
-                        dx(adjusted_x, adjusted_y).at(chosen_channel)
-                        * y_kernel[y_filter][x_filter];
-                    ddyy_i.at(std::integral_constant<int, 0>{}) +=
-                        dy(adjusted_x, adjusted_y).at(chosen_channel)
-                        * y_kernel[y_filter][x_filter];
-                }
-            }
-            ddxx(x, y) = ddxx_i;
-            dxdy(x, y) = dxdy_i;
-            ddyy(x, y) = ddyy_i;
-        }
-    }
-}
-
 /// \brief Generate mean kernel
 /// \ingroup ImageProcessingMath
 ///
 /// Fills supplied view with normalized mean
```

Comment:
Perhaps you could run find and replace for `/// Fills supplied view` changing to `/// Fills kernel`. This could be done even after merge as single commit with `[ci skip]` tag in subject line. No need to CI this kind of change.